src/scss/index.scss

```diff
@@ -1 +1,109 @@
+@import "variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
 
+$enable-shadows: true;
+
+
+.btn {
+  text-decoration: none;
+}
+
+/* nav bottom */
+.navbar-bottom {
+  background-color: $secondary;
+
+  i.fas {
+    color: $primary;
+  }
+
+  p {
+    color: $primary;
+  }
+}
+
+/* body styles */
+body {
+  background-color: $primary;
+}
+
+.bg-img {
+  //image from: https://www.pexels.com/nb-no/bilde/sjarmerende-om-harmoni-ung-4130215/
+  background-image: url("/img/pexels-elina-sazonova-4130215.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+/* Authentication page */
+
+.navbar-custom {
+  background-color: $primary;
```

Comment:
DRY, `.bg-primary`

Reply:
Deleted this class name as I am using .navbar